neuron/src/app/Neuron/CLI/Search.hs

```diff
@@ -23,8 +23,8 @@ searchScriptArgs :: SearchCommand -> [String]
 searchScriptArgs SearchCommand {..} =
   let searchByArgs =
         case searchBy of
-          SearchByTitle -> ["(^# )|(^title: )", "2"]
-          SearchByContent -> ["", "2"]
+          SearchByTitle -> ["(^# )|(^title: )", "2", "/*.{md,org}"]
+          SearchByContent -> ["", "2", "/*.{md,org}"]
```

Comment:
Instead of hardcoding this value (`"/*.{md,org}"`), we should use [`getZettelFormats`](https://github.com/srid/neuron/blob/8d46c5829297ac5a0030b712d9fc74819198b39f/neuron/src/lib/Neuron/Config/Type.hs#L45) to determine it automatically. You can pass the config value to `interactiveSearch` and use it here.

So that if someone has org files lying around, and they are not enabled in neuron.dhall, then `neuron search` should not be searching those. And vice-versa for Markdown files.

Reply:
Thanks, will update.

Reply:
Done!